### PR TITLE
ci: deploy on tag push

### PR DESCRIPTION
**Pull request by rokroskar**
tagged build: https://travis-ci.org/SwissDataScienceCenter/renku-ui/builds/460158643

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -51,8 +51,14 @@ script:
...
 deploy:
-  - provider: script
+  - &script
+    provider: script
     skip_cleanup: true
     script: bash travis-deploy.sh
-    on:
+    on: &script_on
       branch: master
+      repo: SwissDataScienceCenter/renku-notebooks
```
Comment (ableuler) [inline .travis.yml]: Copy-paste error here?

Comment (rokroskar) [reply .travis.yml]: oops

**Comment by ciyer** [comment]
Rebase against master once #357 gets merged. That should fix the ci problems.

**Comment by rokroskar** [comment]
👍 will do

**Comment by rokroskar** [comment]
I have high hopes for https://travis-ci.org/SwissDataScienceCenter/renku-ui/builds/460271085

**Comment by rokroskar** [comment]
@ciyer @ableuler: the tagged build passed so I think this is gtg.

**Pull request merged**